### ISSUE TITLE
'Done' button in the membership requests list

### ISSUE THREAD

**Issue by bum2** [FreedomCoop/valuenetwork#72]
Add a 'Done' button in the membership requests List to unlist each request when done (and perhaps create an 'old requests' tab to keep history)

**Comment by skoria** [comment]
current process is:

1. admins review membership requests
2. the admin assigns themselves a task of dealing with that membership request
3. (at this point) email / telegram exchange to sort out details of request/discuss membership acceptance (in future this would be done with private messages). this admin may need access to the member's profile during this time to update with details gathered. 
4. At this point I'm not sure if the prospective member pays or works for the fee?!
5. the admin clicks the "done" button on their task.

or if that's not it please correct me!

So in summary, regardless of clicking "done", we can currently just keep track on telegram of which requests are being dealt with by who.

**Comment by bhaugen** [comment]
> Add a 'Done' button in the membership requests List

I added a selection widget to say which agent was created for a membership request that takes it off the list. You can see that here now: https://ocp.freedomcoop.eu/accounting/membership-requests/

**Comment by bhaugen** [comment]
@skoria does this satisfy the same need as the done button? And can I close this issue?
